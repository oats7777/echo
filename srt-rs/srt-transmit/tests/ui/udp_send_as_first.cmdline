["udp://127.0.0.1:4000" ,  "udp://127.0.0.1:4000"]