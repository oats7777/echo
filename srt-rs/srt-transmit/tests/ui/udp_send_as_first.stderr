Invalid settings detected: Must not designate a ip to receive UDP. Example: udp://:1234 ,  not udp://127.0.0.1:1234. If you with to bind to a specific adapter ,  use the adapter setting instead.

See srt-transmit --help for more info
